src/index.html

```diff
@@ -1,6 +1,32 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- Google Analytics -->
+    <script>
+      (function(i, s, o, g, r, a, m) {
+        i['GoogleAnalyticsObject'] = r;
+        (i[r] =
+          i[r] ||
+          function() {
+            (i[r].q = i[r].q || []).push(arguments);
+          }),
+          (i[r].l = 1 * new Date());
+        (a = s.createElement(o)), (m = s.getElementsByTagName(o)[0]);
+        a.async = 1;
+        a.src = g;
+        m.parentNode.insertBefore(a, m);
+      })(
+        window,
+        document,
+        'script',
+        'https://www.google-analytics.com/analytics.js',
+        'ga'
+      );
+
+      ga('create', 'UA-62288390-15', 'auto');
+      ga('send', 'pageview');
```

Comment:
doesnt this cause google analytics event to be fired twice? once when index.html executes and once in App.tsx? should we just leave App.tsx one?

Reply:
That's an excellent point - I'll remove this from `index.html`